include/OpenSpaceToolkit/Physics/Data/Manifest.hpp

```diff
@@ -0,0 +1,81 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Physics_Environment_Data_Manifest__
+#define __OpenSpaceToolkit_Physics_Environment_Data_Manifest__
+
+#include <OpenSpaceToolkit/Core/Containers/Array.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Dictionary.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Map.hpp>
+#include <OpenSpaceToolkit/Core/FileSystem/File.hpp>
+#include <OpenSpaceToolkit/Core/Types/String.hpp>
+
+#include <OpenSpaceToolkit/IO/URL.hpp>
+
+#include <OpenSpaceToolkit/Physics/Environment/Ephemerides/SPICE/Kernel.hpp>
+#include <OpenSpaceToolkit/Physics/Time/Instant.hpp>
+
+namespace ostk
+{
+namespace physics
+{
+namespace data
+{
+
+using ostk::core::types::String;
+using ostk::core::ctnr::Array;
+using ostk::core::ctnr::Map;
+using ostk::core::fs::File;
+using ostk::core::ctnr::Dictionary;
+
+using ostk::physics::time::Instant;
+
+/// @brief                      Data class for the OSTk Data Manifest
+
+class Manifest
+{
+   public:
+    /// @brief              Output stream operator
+    ///
+    /// @param              [in] anOutputStream An output stream
+    /// @param              [in] aManifest A manifest
+    /// @return             A reference to output stream
+
+    friend std::ostream& operator<<(std::ostream& anOutputStream, const Manifest& aManifest);
+
+    /// @brief              Return true if manifest is empty
+    ///
+    /// @return             True if manifest is empty
```

Comment:
```suggestion
    /// @brief              Return true if manifest is not empty
    ///
    /// @return             True if manifest is not empty
```
Isn't this the case?

Reply:
Yep, thanks. I renamed the function for alignment and forgot to update here.